README.md

```diff
@@ -5,6 +5,11 @@ Utility methods to create `StreamTransfomer` instances to manipulate Streams.
 Like `asyncMap` but events are buffered in a List until previous events have
 been processed rather than being called for each element individually.
 
+# asyncMapBuffer
```

Comment:
this was already here – just documenting it?

Reply:
Copy/paste/fail to rename. 😳